### DISCUSSION TITLE
`hexo s -o` could not work

### DISCUSSION THREAD

**Discussion by mobeicanyue** [hexojs/hexo#5492] [Question / Answer] [answered]
`hexo s -o` could not work, the sever running normally,
`INFO  Hexo is running at http://localhost:4000/ . Press Ctrl+C to stop.`
 but it couldn't auto open the `chrome` browser.

I using Archlinux, and hexo info like:
```
hexo-cli: 4.3.2
os: linux 6.8.9-arch1-2 Arch Linux
node: 20.13.0
acorn: 8.11.3
ada: 2.7.8
ares: 1.28.1
base64: 0.5.2
brotli: 1.1.0
cjs_module_lexer: 1.2.2
cldr: 44.1
icu: 74.2
llhttp: 8.1.2
modules: 115
napi: 9
nghttp2: 1.61.0
openssl: 3.3.0
simdutf: 5.2.4
tz: 2023c
undici: 6.13.0
unicode: 15.1
uv: 1.48.0
uvwasi: 0.0.20
v8: 11.3.244.8-node.20
zlib: 1.3.1
```

How can I fix it?

**Comment by mobeicanyue** [comment]
I don't know why hexo doesn't recognize my Chrome

**Comment by uiolee** [comment]
show full log

**Reply by mobeicanyue** [reply]
Haven't log yet on console, how the hexo calling the browser?

**Comment by uiolee** [comment]
https://github.com/hexojs/hexo-server/blob/ffc8743ecae5eb21bed60130293177d300113592/lib/server.js#L30-L32

**Comment by mobeicanyue** [comment, marked as answer]
It seems like `open` module's problem, https://github.com/sindresorhus/open/issues/343
